libs/screen/image.ts

```diff
@@ -24,6 +31,24 @@ namespace image {
         }
         return r
     }
+
+    /**
+     * Returns the width or height of a picture.
+     *
+     * @param picture The picture to get the width or height of
+     * @param dimension The dimension to get
+     * @returns
+     */
+    //% blockId=image_get_dimension
+    //% group="Create"
+    //% blockNamespace="images"
+    //% block="$picture $dimension"
+    //% picture.shadow=variables_get
+    //% picture.defl=picture
```

Comment:
Should the value `picture` be quoted here? That's how it appears where it's been added elsewhere in the PR, e.g.: `//% picture.defl="picture"`

Reply:
it isn't required! only required if there is a space in the name